hardhat.config.ts

```diff
@@ -4,6 +4,7 @@ import "hardhat-deploy";
 import dotenv from "dotenv";
 import yargs from "yargs";
 import { getSingletonFactoryInfo } from "@safe-global/safe-singleton-factory";
+import "@nomicfoundation/hardhat-ethers";
```

Comment:
This package is imported in the hardhat-toolbox. Please remove this import statement and the dependency

Reply:
Same for ethers